_community_projects/diggie.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Diggie
+description: Professional GUI client for AWS OpenSearch, desktop app of Mac / Windows that makes operating with OpenSearch easier. This project is currently a developer preview version. Feedbacks and opinions are welcome.
+owner: Serverless Operations, Inc.
+owner_link: https://serverless.co.jp
+link: https://github.com/DiggieApp/Diggie
+download_link: https://github.com/DiggieApp/Diggie/releases/tag/0.1.0
+license: Diggie EULA (developer preview)
+license_link: "#" # in preparation
```

Comment:
I'd rather this lead to a 404 than to `#`. If you know the URL where your license will be already, then just put it in now and we don't have to change it later.

Reply:
@stockholmux I get it 🙌  Added license_link and we'll make the link available soon!